iocage/cli/create.py

```diff
@@ -98,13 +100,14 @@ def cli(release, template, count, props, pkglist, basejail, empty, short,
                 "message": "Template creation only supports TAGs!"
             })
         except ValueError:
-            ioc_common.logit({
-                "level"  : "WARNING",
-                "message": "This may be a short UUID, template creation only"
-                           " supports TAGs"
-            })
-            if not click.confirm("\nProceed?"):
-                exit()
+            if not force:
+                ioc_common.logit({
+                    "level"  : "WARNING",
+                    "message": "This may be a short UUID, template creation only"
```

Comment:
my personal preference is to have error messages in-tact, to make grepping for them easier
failing that, i think the next best would be to break on `,` in this case.